br/pkg/lightning/config/bytesize_test.go

```diff
@@ -89,11 +89,11 @@ func TestByteSizeTOMLDecode(t *testing.T) {
 		},
 		{
 			input: "x = ['100000']",
-			err:   "toml: cannot load TOML value.*",
+			err:   "toml: incompatible types:.*",
```

Comment:
Some linter needs to upgrade the version of toml. so it change some error message.